### PR TITLE
Development 313 offline nodes notifacation

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
### Description

- Added a notification helper that will notify the user every 15 minutes if there are deployments on an offline node.
- Updated the toast warning background, icon color, and text color in dark/light mode.   

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1220

### Screenshots

- image

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/utils/custom_toast.ts [hunk 1]
```diff
@@ -27,4 +38,20 @@ export function createCustomToast(content: string, type: ToastType) {
   };
 
   createToast(content, toastOptions);
+
+  setTimeout(() => {
+    // Change the colors of the icons/contents of all opened toast to be `tonal` as vuetify.
+    const leftIcons = document.getElementsByClassName("mosha__icon");
+    const closIcons = document.getElementsByClassName("mosha__toast__close-icon");
```
Comment (AhmedHanafy725) [inline packages/playground/src/utils/custom_toast.ts:45]:
```suggestion
    const closeIcons = document.getElementsByClassName("mosha__toast__close-icon");
```

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: packages/playground/src/components/tf_notification.vue [hunk 2]
```diff
@@ -5,22 +5,57 @@
...
 const profileManager = useProfileManager();
 const contractsCount = ref(0);
 
+async function isThereAnOfflineNodes(grid: GridClient | null) {
+  const deploymentsNodeIds: number[] = [];
+  const nodesWithPubIps: number[] = [];
+
+  const offlineNodes: NodeInfo[] = await getOfflineNodes(grid);
+  const userContracts: any = await grid!.contracts.listMyContracts();
+  userContracts.nodeContracts.map((contract: any) => {
```
[the remark this answers is not in the document]
Comment (Mahmoud-Emad) [reply packages/playground/src/components/tf_notification.vue:27]: Which type should be used?

**Review by AhmedHanafy725 — APPROVED** [review]

**Review by 0oM4R — CHANGES_REQUESTED** [review]
Screenshot from 2023-10-24 13-37-57

looks like we have an issue with the colors in dark mode can you check this ?

**Review by 0oM4R — APPROVED** [review]
image
image

Great work!

**Comment by 0oM4R** [comment]
i beli

> Screenshot from 2023-10-24 13-37-57
> 
> looks like we have an issue with the colors in dark mode can you check this ?

it was cached somehow worked fine after that

**Pull request merged**